addons/cert-manager/metadata.yaml

```diff
@@ -14,5 +14,5 @@ dependencies:
   - name: fluxcd
 
 system:
-  vela: ">=1.6.0-alpha.1"
+  vela: ">=v1.7.0-alpha.1"
```

Comment:
does the requirements a must have?

Reply:
yes, the `take over` policy is added in `1.7.0`.